exercises/practice/grade-school/.meta/src/GradeSchool.example.elm

```diff
@@ -35,6 +44,8 @@ studentsInGrade grade school =
             []
 
 
-allStudents : School -> List ( Grade, List Student )
+allStudents : School -> List Student
```

Comment:
Does this rely on the fact that Dict.toList returns a list sorted by the key? If so probably worth a comment saying so ...

Reply:
Yes it does. It's clear on the documentation, so it should be reliable.
I added a comment.